<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit</class>
 <widget class="QDialog" name="credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(0, 0, 0);
	border-radius: 5px;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(100, 100, 100);
  border: 2px gray;
  font: 8pt &quot;Comic Sans MS&quot;;
  border-radius: 10px;
color: rgb(239, 239, 239);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_sum">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  font: 10pt &quot;Comic Sans MS&quot;;
  background-color: rgb(100, 100, 100);
  qproperty-alignment: 'AlignVCenter';
  border: 2px gray;
  border-radius: 10px;
  color: rgb(239, 239, 239);
}
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_percent">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  font: 10pt &quot;Comic Sans MS&quot;;
  background-color: rgb(100, 100, 100);
  qproperty-alignment: 'AlignVCenter';
  border: 2px gray;
  border-radius: 10px;
  color: rgb(239, 239, 239);
}
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_term">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  font: 10pt &quot;Comic Sans MS&quot;;
  background-color: rgb(100, 100, 100);
  qproperty-alignment: 'AlignVCenter';
  border: 2px gray;
  border-radius: 10px;
  color: rgb(239, 239, 239);
}
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_result">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(243, 114, 32);
	font: 75 10pt &quot;Comic Sans MS&quot;;
	border: 2px gray;
	border-radius: 10px;
	color: rgb(0, 0, 0);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Diff">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Comic Sans MS&quot;;
		color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Ann">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Comic Sans MS&quot;;
		color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="output_monthly">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  font: 10pt &quot;Comic Sans MS&quot;;
  background-color: rgb(100, 100, 100);
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 2px gray;
  border-radius: 10px;
  color: rgb(239, 239, 239);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="output_percent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  font: 10pt &quot;Comic Sans MS&quot;;
  background-color: rgb(100, 100, 100);
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 2px gray;
  border-radius: 10px;
  color: rgb(239, 239, 239);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="output_total">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  font: 10pt &quot;Comic Sans MS&quot;;
  background-color: rgb(100, 100, 100);
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 2px gray;
  border-radius: 10px;
  color: rgb(239, 239, 239);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Ежемесячный платёж</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		font: 10pt &quot;Comic Sans MS&quot;;
		color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Начисленный процент</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Общая сумма</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
